.gitignore

```diff
@@ -25,6 +25,10 @@
 /web/sites/simpletest
 /web/tmp
 
+# Ignore database dumps
+*.sql
+*.sql.gz
```

Comment:
Perhaps even `*.gz`?

Reply:
I just copied it from another project. Maybe in some cases compressed archives need to be in the repository? 🤷